.gitignore

```diff
@@ -136,7 +138,7 @@ android/firebase-app-distribution.json
 android/firebase-test-lab.json
 android/huawei-appgallery.json
 android/res/xml/network_security_config.xml
-server
+./server/
```

Comment:
what is the difference?

Reply:
`server` ignores folder ewerywhere in repo
`./server` ignores folder only in the root